src/api/settings/getServerSettings.js

```diff
@@ -34,6 +34,8 @@ export type ApiResponseServerSettings = {|
   realm_icon: string,
   realm_name: string,
   realm_uri: string,
+  // realm_web_public_access_enabled added for server v5.0
+  realm_web_public_access_enabled?: boolean,
```

Comment:
```suggestion
  // TODO(server-5.0) realm_web_public_access_enabled added at FL 116
  realm_web_public_access_enabled?: boolean,
```

For details and rationale, see: https://github.com/zulip/zulip-mobile/blob/main/docs/style.md#todo-server